apis/validator/beacon_committee_subscriptions.v2.yaml

```diff
@@ -0,0 +1,61 @@
+post:
+  operationId: "prepareBeaconCommitteeSubnetV2"
+  summary: Signal beacon node to prepare for a committee subnet
+  description: |
+    After beacon node receives this request:
+    - Search using discv5 for peers related to this subnet and replace current peers with those ones if necessary. 
+    - Check each provided `slot_signature`, to determine whether any validator has been [assigned the duty of aggregation](https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/validator.md#aggregation-selection)
+    based on their slot signature.
+  tags:
+    - ValidatorRequiredApi
+    - Validator
+  requestBody:
+    content:
+      application/json:
+        schema:
+          title: SubscribeToBeaconCommitteeSubnetRequestBody
+          type: array
+          items:
+            type: object
+            properties:
+              validator_index:
+                $ref: '../../beacon-node-oapi.yaml#/components/schemas/Uint64'
```

Comment:
From https://ethresear.ch/t/distributed-validator-middlewares-and-the-aggregation-duty/13044/5:
> This and most endpoints in the beacon API are designed to support asking about multiple validators simultaneously. We would return one boolean for each validatorIndex we query on behalf of. Each boolean indicates whether that validator must aggregate or not.

`validator_index` is an integer here, which means you will be able to specify only a single validator. As such returning an array doesn't make sense.

Reply:
Yes but `validator_index` is an integer field within an object and that object is wrapped in an array. My understanding is a request body like this would be perfectly valid and would represent multiple validators:

```json
[
  {
    "validator_index": "1",
    "committee_index": "1",
    "committees_at_slot": "1",
    "slot": "1",
    "slot_signature": "0x123...f"
  },
  {
    "validator_index": "2",
    "committee_index": "1",
    "committees_at_slot": "1",
    "slot": "1",
    "slot_signature": "0x423...f"
  },
  {
    "validator_index": "3",
    "committee_index": "1",
    "committees_at_slot": "1",
    "slot": "1",
    "slot_signature": "0x323...f"
  }
]
```

In this scenario, the response could look like:
```json
[
  { "is_aggregator": false},
  { "is_aggregator": false},
  { "is_aggregator": true}
]
```

Reply:
You are absolutely right.